src/fal/cli/dbt_runner.py

```diff
@@ -73,15 +72,26 @@ def dbt_run(args, models_list: List[str], target_path: str, run_index: int):
     if args.selector:
```

Comment:
Shouldn't we remove this too? I think a selector is processed in the fal flow level and then we pass the correct `--select` flags to dbt.

Reply:
yea i am not sure how we can support selectors right now seems like the currently code probably doesn't handle it perfectly either.. for example we dont handle tags.. ok creating a ticket for it

Reply:
I am not sure it doesn't work, I haven't looked too much in detail inside fal flow selection flags process. I think it _should_ work? Do you want to add an integration test to test it?

Reply:
Adding tests in a different PR..